Makefile

```
@@ -221,6 +225,33 @@ build-observability-dashboard: ## Build observability dashboard image
 		-t $(OBSERVABILITY_DASHBOARD_IMAGE) .
 	@echo "$(COLOR_GREEN)✓$(COLOR_RESET) Observability dashboard built: $(OBSERVABILITY_DASHBOARD_IMAGE)"
 
+build-credential-sidecars: build-credential-github build-credential-jira build-credential-k8s build-credential-google ## Build all credential sidecar images
```

checkmake (0.3.2)

[warning] Target "build-credential-sidecars" should be declared PHONY.

(phonydeclared)